packages/ai-bot/lib/matrix.ts

```diff
@@ -132,6 +122,54 @@ export async function sendError(
   }
 }
 
+export const toMatrixMessageContent = (
+  functionCall: FunctionToolCall,
+  eventToUpdate: string | undefined,
+): IContent | undefined => {
+  let { arguments: payload } = functionCall;
+  if (functionCall.name === 'patchCard') {
```

Comment:
use if/else for the all commands is not sustainable way to do this. imagine if we have 10 different commands. even a `case` statement is probably not a good idea. I would hope that we have some `Command` class (or maybe card) and that there is a serialize method on it so that we could serialize the command for sending as a matrix message

Reply:
Appreciate the pushback here. 

We can decouple here to aim for no checks for `patchCard` and `searchCard` shud ever exist in the ai bot. Let me experiment with this in my original PR to see how viable this is. 

the ai bot essentially only looks for tool calls and forwards the tool calls to the matrix message without any notion of the domain of the command. Any bad responses of the ai bot is handled from the consumer of these matrix messages. 

This also means some of the comments about stronger types may not be applicable since the ai bot doesn't have to know it.